webapp/src/main/resources/public/js/components/workbench/TextUnit.js

```diff
@@ -805,18 +813,28 @@ let TextUnit = createReactClass({
         let assetPathWithGitInfoTooltip =
             <Tooltip id="{this.props.textUnit.getId()}-gitInfo">{this.props.intl.formatMessage( {id: 'workbench.gitBlameModal.info'} )}</Tooltip>;
 
+        let assetPathTranslationHistoryTooltip =
+            <Tooltip id="{this.props.textUnit.getId()}-translation-history">{this.props.intl.formatMessage( {id: 'workbench.translationHistoryModal.info'} )}</Tooltip>;
```

Comment:
I'm looking at the current code: `"{this.props.textUnit.getId()}-gitInfo"` and it doesn't seem to work. So I suspect this doesn't work either. Could you fix both?

Reply:
Done

Reply:
thanks!